db_fill_minimum.py

```diff
@@ -0,0 +1,170 @@
+import os
```

Comment:
was tut die file, dass sie extra erstellt werden musste? macht es nicht sinn sie in db_fill zu packen?

Reply:
Damit werden nur notwendige Tabellen erstellt (2 Stück). Alles andere kann dann im Frontend created werden. 
Im Vergleich dazu werden bei db_bill alle Tabellen mit irgendwas befüllt.